tests/src/Interop/StringMarshalling/AnsiBSTR/PinvokeDef.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.InteropServices;
+using System;
+using System.Reflection;
+using System.Text;
+
+#pragma warning disable CS0618
+
+namespace NativeDefs
+{
+    public static class PInvokeDef
+    {
+        public const string NativeBinaryName = "AnsiBStrTestNative";
```

Comment:
I really like the `nameof()` keyword for handling this pattern. Make the name of the class `public static class AnsiBStrTestNative` and then update the attribute to `[DllImport(nameof(AnsiBStrTestNative))]`.